.gradle/buildOutputCleanup/cache.properties

```diff
@@ -0,0 +1,2 @@
+#Tue Mar 14 09:13:05 NOVT 2023
+gradle.version=7.5.1
```

Comment:
You have added .idea and gradle files before gitignore, as i see

You should remove it from your index, see `git rm --cached`

Reply:
fixed